### ISSUE TITLE
Unable to create horizontal stacked bar chart

### ISSUE THREAD

**Issue by vince92079** [Megabit/Blazorise#2294]
I'm trying to create a Horizontal Stacked Bar chart but after settings the Stacked option in Axes the chart is still showing it's chart as a non-stacking bar chart.  My options are setup as:

```cs
private BarChartOptions barchartOptions = new BarChartOptions
    {
        
        Scales = new Scales
        {
            XAxes = new List<Axis> {
                new Axis
                {
                    Stacked = true,
                    Display = true
                }
            },
            YAxes = new List<Axis> {
                new Axis
                {
                    Stacked = true,
                    Display = true
                }
            },
        },
        Legend = new Legend
        {
            Display = true
        }
    };
```
    
I can change the `Display` options in the Legend, XAxes and YAxes and the bar chart will update so I know the options do work with it.

I also tried converting the bar chart options to using a `OptionsObject` and the problem with the stacking was still there.

My html code is 
    
`<HorizontalBarChart @ref="HorizontalChart" TItem="int" Options="@barchartOptions"/>`

I also had to add in the fix listed here to get the bars to show up https://github.com/stsrki/Blazorise/issues/2084

**Comment by stsrki** [comment]
I'm unable to test now, but please check these two tickets. They might help you

- https://github.com/stsrki/Blazorise/issues/2084
- https://github.com/stsrki/Blazorise/issues/2270

**Comment by vince92079** [comment]
I originally found those issues yesterday because I was unable to get the bars to appear.  As mentioned above I have already implemented that change and added `Type = "horizontalBar"` to my Dataset and the bars do appear on the chart.  

I cannot get the bars to stack even though I've set the `Stacked` options to true for both Axis per my code in the original post.  The Bar Chart on my page shows up as this.  I would expect the three values to be stacked based on what I've set the options to.

![BarSetup](https://user-images.githubusercontent.com/1072676/117043617-78d56b80-acdb-11eb-9567-cbabb864483a.png)
